src/common/test/MockApolloClientProvider.tsx

```diff
@@ -23,8 +25,13 @@ export function MockApolloClientProvider({
   responseProvider: ResponseProvider;
 }): React.ReactElement {
   const customGlobal = global as GlobalWithFetchMock;
-  customGlobal.fetchMock.mockResponse((variables?: RequestVariables) => {
-    const response = createMockedProfileResponse(responseProvider(variables));
+  customGlobal.fetchMock.mockResponse(async (req: Request) => {
```

Comment:
Does this mock all fetch calls? Is there a risk that this mock stays on for some tests that would not wish to mock fetch?

Reply:
Yes, it mocks all calls - but only when the test includes the MockApolloClientProvider.  And only Apollo graphQL-queries are fetched in the UI. It is true that other fetches are mocked too, but at the moment there aren't any and then MockApollo should not be used. Is the risk too big?

Reply:
Probably not a big risk, just wondering if it could cause problems in the future where developer would not notice why is his fetch mocked if he tries testing code with other fetches.

Reply:
I'll change this to catch only /graphql/ paths.

Reply:
Done.